### PR TITLE
Update channels for 1.18

### PR DESCRIPTION
**Pull request by mikesplain**

### CODE REVIEW

File: channels/stable [hunk 1]
```diff
@@ -55,6 +55,9 @@ spec:
     networking:
       kubenet: {}
   kubernetesVersions:
+  - range: ">=1.18.0-alpha.1
+    recommendedVersion: 1.18.0-beta.1
+    requiredVersion: 1.18.0-alpha.1
```
Comment (rifelpet) [inline channels/stable:60]: should these be:
```suggestion
    recommendedVersion: 1.18.3
    requiredVersion: 1.18.0
```
?

Comment (mikesplain) [reply channels/stable:60]: Doh, thank you

Comment (mikesplain) [reply channels/stable]: Fixed thanks. Too much on the clipboard.

**Pull request merged**